requirements.txt

```diff
@@ -1,4 +1,5 @@
 Babel==2.3.4
+chardet==3.0.4
```

Comment:
@xmo-odoo looks good to me from the packaging point of view.
The latest ubuntu packages for p2 and p3 are at version 3.0.4 as required.
Fedora 27 and 28 are at the same version.